src/hooks/useLogout.tsx

```diff
@@ -29,6 +30,7 @@ export const useLogout = (): LogoutHook => {
       setIsLoggingOut(true);
       await clearAuthInfo();
       setProducts([]);
+      setFeatures({} as Feature);
```

Comment:
Should it be the plural, `Features`?

Reply:
yeah i think it should be haha good catch!